test/acceptance/framework/environment.go

```diff
@@ -87,22 +91,22 @@ func (k kubernetesContext) KubernetesClient(t *testing.T) kubernetes.Interface {
 	configPath, err := k.KubectlOptions().GetConfigPath(t)
 	require.NoError(t, err)
 
-	t.Logf("Creating client from config path at %s for context %s", configPath, k.contextName)
-	config, err := k8s.LoadApiClientConfigE(configPath, k.contextName)
+	t.Logf("Creating client from config path at %s for context %s", configPath, k.kubeContextName)
+	config, err := k8s.LoadApiClientConfigE(configPath, k.kubeContextName)
 	require.NoError(t, err)
 
 	client, err := kubernetes.NewForConfig(config)
 	require.NoError(t, err)
 
-	k.client = client
+	k.client = helpers.KubernetesClientFromOptions(t, k.KubectlOptions())
```

Comment:
Is there a difference between these two client? Can we get away with using the client from L98 here? That way, we could make `helpers.KubernetesClientFromOptions` package private

Reply:
🤦‍♀️ they are the same, but the goal was to refactor this to helpers. So all the code above line 101 should not be there. Let me update.

Reply:
Using the helper package makes more sense!! 🚢

Reply:
Updated!